prow/oss/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -0,0 +1,219 @@
+presubmits:
+  GoogleCloudPlatform/oss-test-infra:
```

Comment:
Shouldn't these just be moved from elsewhere?

Reply:
Sorry but I'm not sure if I understand the question

Reply:
Can we `git mv` these files instead of copying them?

Reply:
I would love to, but it'll fail `pull-prow-config-validate` job

Reply:
How about if we avoid moving the prowjobs? I'd rather they not be under the same dir as the prow cluster config anyway

Reply:
Even if we don't move the prowjobs this PR would still fail, as `pull-prow-config-validate` also checks plugins and core configs. 

Also, it will be a little bit confusing which prow the jobs belong to when it becomes
```
- prow
  - oss
  - knative
  - jobs
```

Reply:
If we are moving things around I really do not want jobs under prow/oss

I wouldn't find that arrangement particularly confusing. I'd also be fine with /prowjobs or /jobs instead of /prow/jobs

Reply:
Ack, updated